plugins/org.python.pydev.ast/src/org/python/pydev/ast/interpreter_managers/InterpreterInfo.java

```diff
@@ -1896,8 +1896,15 @@ public void setStringSubstitutionVariables(Properties stringSubstitutionOriginal
     }
 
     @Override
-    public Properties getStringSubstitutionVariables() {
-        return this.stringSubstitutionVariables;
+    public Properties getStringSubstitutionVariables(boolean addEnvironmentVaiables) {
+        Properties ret = new Properties();
```

Comment:
if `addEnvironmentVariables == false`, just return `this.stringSubstitutionVariables` directly (instead of doing a copy).